Sources/dump-unified-graph/DumpUnifiedGraph.swift

```diff
@@ -0,0 +1,109 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import ArgumentParser
+import SymbolKit
+
+@main
+struct DumpUnifiedGraph: ParsableCommand {
+    @Option(help: "module's symbol graph to output (will infer a single module, but will fail if multiple modules are being loaded)")
```

Comment:
This is just a nit-pick, but I think it's kind of a slick feature of ArgumentParser.

If you replace this with an explicit [`ArgumentHelp(_:discussion:)`](https://apple.github.io/swift-argument-parser/documentation/argumentparser/argumenthelp/init(_:discussion:valuename:visibility:)) initializer you can put the more descriptive text in a discussion section:

```suggestion
    @Option(help: ArgumentHelp("module's symbol graph to output", description: "will infer a single module, but will fail if multiple modules are being loaded")
```